### ISSUE TITLE
No module core_f90 

### ISSUE THREAD

**Issue by ghost** [oseledets/ttpy#23]
After installation the last version if you try to do "import tt", you'll get the following message:
`Traceback (most recent call last):<
  File "<stdin>", line 1, in <module>
  File "tt/__init__.py", line 12, in <module>
    from core.tt import *
  File "tt/core/tt.py", line 4, in <module>
    from matrix import matrix
  File "tt/core/matrix.py", line 3, in <module>
    import core.core_f90 as _core_f90
ImportError: No module named core.core_f90`

If you roll back to the version from 4th of May, everything will be OK.
OS: Linux Ubuntu 16

**Comment by daskol** [comment]
@RerRayne How did you install the package?

**Comment by RerRayne** [comment]
I cloned repository by 
`git clone https://github.com/oseledets/ttpy.git`
So I have understood the problem. After 
`python setup.py install`
I started python from ttpy directory and when I did import, python tried to found core_f90 in ./tt/core in a local dir.
If you move to another directory, everything will be ok. 
So, the better way to avoid this kind of stupid problem is using pip installation:
`pip install git+https://github.com/oseledets/ttpy` 
against downloading and compilation.
